support/mender-device-identity

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+# Example script called by Mender agent to collect device identity
+# data. The script needs to be located at
+# $(bindir)/mender-device-indentity path for the agent to find it.
+# The script shall exit with non-0 status on errors. In this case the
+# agent will discard any output the script may have produced.
+#
+# The script shall output identity data in <key>=<value> format, one
+# entry per line. Example
+#
+# $ ./mender-device-identity
+# mac=de:ad:ca:fe:00:01
+# cpuid=1112233
```

Comment:
Btw, will we pay any attention to order? We should perhaps keep it sorted in the client to make sure it doesn't mess up the hash.